README.md

```diff
@@ -94,26 +121,6 @@ Dor.find("druid:pv820dk6668").destroy
 %w[pv820dk6668 rn653dy9317 xb482bw3979 hj185vb7593 hv992ry2431].each{ |pid| Dor.find("druid:#{pid}").destroy }
 ```
 
-### virtual-merge script
-
-NOTE: temporary until bulk async action implemented in UI
-
-Merges multiple image objects into a parent object via virtual object structure (e.g. an atlas in which each map has been individually cataloged).  Creates a virtual composite object with contentMD pointing to binaries held in other objects; the child objects have an isConstituentOf element in RELS-EXT allowing traversal to a parent object in discovery interfaces.
-
-virtual merge can be run from the Argo application root directory:
-
-```bash
-RAILS_ENV=production bundle exec ./bin/virtual-merge --purge druid:pp000pp0000 druid:cc111cc1111 druid:cc222cc2222
-```
-
-where `pp000pp0000` is the druid for the virtual composite object and the child objects are `cc111cc1111` and `cc222cc2222`.  More info, including object requirements: <https://consul.stanford.edu/display/chimera/Virtual+Object+Combinator+I>
-
-For help:
-
-```bash
-bundle exec ./bin/virtual-merge -h
-```
-
```

Comment:
I would prefer we remove this as a last step, after all the virtual merge pieces are there and we're satisfied.  I can make a separate ticket, or you can just make a PR with "[blocked by virtual merge bulk action]" or some such.

Reply:
:+1: to making a separate ticket to tearing out the CLI after the UI is confirmed sufficient. I'll restore that stuff in this PR.